<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmUpgrade</class>
 <widget class="QWidget" name="frmUpgrade">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="updateBtn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始升级</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="filePathEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>104</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="updateProgressBar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="select_pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
